applications/DEMApplication/custom_elements/spheric_particle.cpp

```diff
@@ -1103,6 +1107,9 @@ void SphericParticle::ComputeBallToRigidFaceContactForce(SphericParticle::Partic
                                      GlobalElasticContactForce, ViscoDampingLocalContactForce, cohesive_force, r_elastic_force,
                                      r_contact_force, mNeighbourRigidFacesElasticContactForce[i], mNeighbourRigidFacesTotalContactForce[i]);
 
+            // Store contact information needed for other processes
+            StoreBallToRigidFaceContactInfo(r_process_info, data_buffer, wall, GlobalContactForce, sliding);
```

Comment:
Same comment. It seems like forces due to rotation can't be stored if we call the method at this location.

Reply:
Same as above.